wgengine/magicsock/magicsock.go

```diff
@@ -83,6 +83,8 @@ var (
 	// on mobile devices, lowers the shutdown interval, and logs more
 	// verbosely about idle measurements.
 	debugReSTUNStopOnIdle, _ = strconv.ParseBool(os.Getenv("TS_DEBUG_RESTUN_STOP_ON_IDLE"))
+	// debugDisableUDP disables the use of UDP, forcing all peer communication over DERP.
+	debugDisableUDP, _ = strconv.ParseBool(os.Getenv("TS_DEBUG_DISABLE_UDP"))
```

Comment:
This makes it sound like it's all UDP but it's just magicsock as implementation. There's also DNS and netstack forwarding. Rename environment var?

Reply:
PTAL